<?xml version="1.0"?>
<launch>
  <arg name="max_rate"  default="5"/>
  <arg name="scale"     default="0.5"/>
  <arg name="image"     default="/robot/front_rgbd_camera/color/image_raw"/>
  <arg name="transport" default="raw"/>

  <node name="person_detector" pkg="pal_person_detector_opencv" type="pal_person_detector_opencv" output="screen">
    <rosparam command="load" file="$(find pal_person_detector_opencv)/config/tracking_params.yaml" />
    <param name="image" value="$(arg image)"/>
    <param name="rate"  value="$(arg max_rate)"/>
    <param name="scale" value="$(arg scale)"/>
    <param name="transport" value="$(arg transport)"/>
    <!-- <remap from="/person_detector/cmd_vel" to="/robot/move/cmd_vel"/> -->
  </node>

</launch>
